pkgs/applications/version-management/git-and-tools/gfold/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gfold";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "nickgerace";
+    repo = pname;
+    rev = version;
+    sha256 = "0ss6vfrc6h3jlh5qilh82psd3vdnfawf1wl4cf64mfm4hm9dda63";
+  };
+
+  cargoSha256 = "09ywwgxm8l1p0jypp65zpqryjnb2g4gririf1dmqb9148dsj29x2";
+
+  meta = with lib; {
+    description =
+      "CLI tool to help keep track of your Git repositories, written in Rust";
```

Comment:
```suggestion
    description = "A tool to help keep track of your Git repositories, written in Rust";
```

Reply:
Hi @AndersonTorres, thanks for the review. I've changed this with your suggestion at 9a536f4d09f6de919790224a156e110f3312a29b.

Just to help me adapt as a new contributor, was this feedback more oriented around the phrasing or the newline? I have my editor using `nixfmt` by default and `nixpkgs-fmt` instead didn't seem to propose any diffs on the previous version.

Reply:
The newline, primarily. It is weird to look a `description` not in the same line.

I suggested the new phrasing because I thought that the old one would break the 80-column limit. I would even suggest to remove the `, written in Rust`, but I think it would be excessive.

No worries, this is mostly stylistical.